template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/AbstractRoleConfigProvider.java

```diff
@@ -0,0 +1,111 @@
+package com.sequenceiq.cloudbreak.cmtemplate.configproviders;
```

Comment:
This class contains RoleConfig Variables initialization code moved from AbstractVolumeConfigProvider so that the common code can be reused across RdsConfigProviders, VolumeConfigProviderss and so on..